docs/contribute.rst

```diff
@@ -130,7 +130,7 @@ To run all integration tests::
 
 To run a particular test function in a particular integration test file::
 
-    tox -e integration -- pytest tests/integration/test_modelsearch.py -k test_summary_individuals
+    tox -e integration -- tests/integration/test_modelsearch.py -k test_summary_individuals
```

Comment:
Use `::` instead of `-k` here also: `tox -e integration -- tests/integration/test_modelsearch.py::test_summary_individuals`

Reply:
Yes this might be better. Does `::` do exact matches instead of pattern match as `-k`?

Reply:
Just have a quick test by running:
 `tox -e integration -- tests/integration/test_modelsearch.py::test_summary_individuals` and 
`tox -e integration -- tests/integration/test_modelsearch.py -k test_summary_individuals`
Here is the diff:
```4,5c4,5
< integration run-test-pre: PYTHONHASHSEED='587217178'
< integration run-test: commands[0] | pytest -vv tests/integration/test_modelsearch.py::test_summary_individuals
---
> integration run-test-pre: PYTHONHASHSEED='1911634784'
> integration run-test: commands[0] | pytest -vv tests/integration/test_modelsearch.py -k test_summary_individuals
12c12
< collected 1 item                                                                                                                                                                                                              
---
> collected 9 items / 8 deselected / 1 selected                                                                                                                                                                                 
```
When `-k`,  tox will collect all items in the file and select the one we want while `::`, tox will directly collect the item we want.

Reply:
Ok, that is another difference. Then `::` is the way to go in our contribution doc.